omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/openmrs1_8/ModuleAdministrationLinks1_8.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.wrapper.openmrs1_8;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This class represents list of administration links for a module.
+ * These links are provided by AdministrationSectionExt class, extended by appropriate classes in installed modules.
+ * Exposed by AdministrationLinksResource1_8 class.
+ *
+ * @see org.openmrs.module.web.extension.AdministrationSectionExt
+ * @see org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.AdministrationLinksResource1_8
+ */
+public class ModuleAdministrationLinks1_8 implements Serializable {
```

Comment:
What is the use of this class? Why not just use AdministrationSectionExt?

Reply:
Well, of course, we could just use `AdministrationSectionExt` if you want. I thought that creating another class would create another level of abstraction so we wouldn't have to depend so much on the `AdministrationSectionExt` if it changes.

Reply:
The less code we have to maintain, the better.

Reply:
After using `AdministrationSectionExt` as a resource I'm getting multiple test failures, because they can't see the `AdministrationSectionExt` class on their classpath. Should I just add `legacyui-omod` as a dependency for all the submodules of version 2.0+?

Reply:
After adding `legacyui-omod` as a depependy I'm getting an unrelated test failure: 
`PersonResource2_2Test`
`display expected:<[Mr. ]Horatio Test Hornblo...> but was:<[]Horatio Test Hornblo...>`
Do you know why is that happening?

Reply:
I think we could still copy the `AdministrationSectionExt` into `omod-common` module and then we wouldn't have to depend on `legacyui-omod` :)

Reply:
Not a good thing to copy a class all over.

Reply:
Commit and i take a look at the error log.

Reply:
https://github.com/openmrs/openmrs-module-webservices.rest/pull/484/commits/c0936b59f565de0a6887dcf99277c9fc8e9a4cd9